### PR TITLE
fix(Sidepanel): prevent icon from moving in large mode

### PR DESCRIPTION
**Pull request by romainseb**
**What is the problem this PR is trying to solve?**
Icons are moving when the sidepanel is in large mode & you collapse and expand.
**What is the chosen solution to this problem?**
Ajust padding to prevent this moves

**Please check if the PR fulfills these requirements**
- [x] The PR commit message follows our [guidelines](https://github.com/talend/tools/blob/master/tools-root-github/CONTRIBUTING.md)
- [ ] Tests for the changes have been added (for bug fixes / features)
- [ ] Docs have been added / updated (for bug fixes / features)
- [ ] Related design / discussions / pages (not in jira), if any, are all linked or available in the PR

**[ ] This PR introduces a breaking change**

### CODE REVIEW

File: output/components.sasslint.txt [hunk 1]
```diff
@@ -15,5 +15,9 @@ src/Drawer/Drawer.scss
 src/FilterBar/FilterBar.scss
   47:6  warning  Vendor prefixes should not be used  no-vendor-prefixes
 
-✖ 4 problems (1 error, 3 warnings)
+src/SidePanel/SidePanel.scss
+  119:10  error  Qualifying elements are not allowed for class selectors  no-qualifying-elements
+  126:10  error  Qualifying elements are not allowed for class selectors  no-qualifying-elements
+
+✖ 6 problems (3 errors, 3 warnings)
```
Comment (nicomaligne) [inline output/components.sasslint.txt]: Some linter errors here ;)

File: packages/components/src/SidePanel/SidePanel.scss [hunk 2]
```diff
@@ -99,52 +99,38 @@ $toggle-button-padding: $padding-small - 1;
...
+	.toggle-btn {
+		margin-bottom: $padding-normal + $padding-large;
+		text-align: left;
+		padding-left: $toggle-button-padding;
 
-			.tc-side-panel-list-item {
-				width: $tc-side-panel-large-docked-width;
-			}
+		:global(.btn.btn-link) {
+			outline: none;
```
Comment (frassinier) [inline packages/components/src/SidePanel/SidePanel.scss]: why?

File: packages/components/src/SidePanel/SidePanel.scss [hunk 3]
```diff
@@ -99,52 +99,38 @@ $toggle-button-padding: $padding-small - 1;
...
 			}
 		}
-	}
 
-	.link {
-		text-align: left;
-		font-size: $font-size-small;
-		text-transform: uppercase;
+		:global(.btn.btn-link) > span {
+			display: none;
```
Comment (frassinier) [inline packages/components/src/SidePanel/SidePanel.scss]: Can we prefer visibility: hidden?

File: packages/components/src/SidePanel/SidePanel.scss [hunk 4]
```diff
@@ -173,38 +160,52 @@ $toggle-button-padding: $padding-small - 1;
...
 	}
 
-	.toggle-btn {
-		margin-bottom: $padding-normal + $padding-large;
-		text-align: left;
-		padding-left: $toggle-button-padding;
+	&.large {
+		width: $tc-side-panel-large-width;
+		> ul {
+			width: $tc-side-panel-large-width;
```
Comment (frassinier) [inline packages/components/src/SidePanel/SidePanel.scss]: Duplication is weird

**Pull request merged**